components/cli/cmd/cellery/setup_create_local.go

```diff
@@ -19,20 +19,32 @@
 package main
 
 import (
-	"github.com/spf13/cobra"
-
+	"fmt"
 	"github.com/cellery-io/sdk/components/cli/pkg/commands"
+	"github.com/spf13/cobra"
 )
 
 func newSetupCreateLocalCommand() *cobra.Command {
+	var isOutput bool
+	var addGlobalGW bool
+	var addObservability bool
 	cmd := &cobra.Command{
 		Use:   "local",
 		Short: "Create a local Cellery runtime",
 		Args:  cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if addObservability && !addGlobalGW {
+				return fmt.Errorf("You can't deploy observability portal without the Global gateway")
```

Comment:
Error message should be updated (ex: Observability portal cannot be deploy deployed... )

Reply:
This check will be removed with new setup stages.